### PR TITLE
D/add box shadow for buttons

### PR DESCRIPTION
**Pull request by jhc19**
- added box shadows to the buttons and level selector.
- the margin of the marker-info box is adjusted to be able to render the box-shadow properly.
- reverted the values of the button outline border back to its original value.

### CODE REVIEW

File: stem-explorer-ng/src/styles.scss [hunk 1]
```diff
@@ -48,7 +48,7 @@ h3, h4, h5, h6, b, strong {
   margin-bottom: 0.2rem;
 }
 h3 {
-  margin-top: 0;
```
Comment (kellymarjorie) [inline stem-explorer-ng/src/styles.scss:51]: Can you add this as a separate class for the H3 it's needed on? as now this margin is on every H3 we have (and we use it a lot :) ) and can you define it as 5px or which one it needs to be?

**Review by kellymarjorie — CHANGES_REQUESTED** [review]

File: stem-explorer-ng/src/styles.scss (hunk 1 above)
Comment (jhc19) [reply stem-explorer-ng/src/styles.scss:51]: Hi Kelly,

I've sorted that just moments ago. 👍

File: stem-explorer-ng/src/app/shared/components/challenge-title/challenge-title.component.scss [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 @mixin title-color($color) {
     color: $color;
+    margin-top: 15px; /*In order to fix the spacing between the level selector and h3 in challenge view*/
```
Comment (kellymarjorie) [inline stem-explorer-ng/src/app/shared/components/challenge-title/challenge-title.component.scss:5]: This `margin-top` needs to be added to a class unique to that element, as this is giving all titles that use this mixin that space at the top (in challenge dialog, list view, etc.)

**Review by kellymarjorie — CHANGES_REQUESTED** [review]

File: stem-explorer-ng/src/app/shared/components/challenge-title/challenge-title.component.scss (hunk 2 above)
Comment (jhc19) [reply stem-explorer-ng/src/app/shared/components/challenge-title/challenge-title.component.scss]: Hi Kelly,

I've made some changes, but I wasn't sure if it still causes the same issue and if it is written right.

**Review by kellymarjorie — APPROVED** [review]

**Pull request merged**